metricbeat/module/elasticsearch/index_summary/data.go

```diff
@@ -91,6 +164,14 @@ func eventMapping(r mb.ReporterV2, info elasticsearch.Info, content []byte) erro
 		return errors.Wrap(err, "failure applying stats schema")
 	}
 
+	var event mb.Event
```

Comment:
This code block moving wasn't necessary at all. It just helped me when comparing the code with `data_xpack.go`